sefaria/model/text.py

```diff
@@ -386,8 +385,10 @@ def alt_titles_regex(self, lang):
         alt_titles = list(map(re.escape, full_title_list))
         reg = '(?P<title>' + '|'.join(sorted(alt_titles, key=len, reverse=True)) + r')($|[:., ]+)'
         try:
-            reg = re.compile(reg, max_mem=384 * 1024 * 1024)
-        except TypeError:
+            options = re.Options()
```

Comment:
Is this going to be something that winds up repeating over and over in the code?